bench/commands/update.py

```diff
@@ -23,7 +23,7 @@
 @click.option('--reset', is_flag=True, help="Hard resets git branch's to their new states overriding any changes and overriding rebase on pull")
 def update(pull, patch, build, bench, requirements, restart_supervisor, restart_systemd, no_backup, force, reset):
 	from bench.utils import update
-	update(pull=pull, patch=patch, build=build, bench=bench, requirements=requirements, restart_supervisor=restart_supervisor, restart_systemd=restart_systemd, backup= not no_backup, force=force, reset=reset)
+	update(pull=pull, patch=patch, build=build, bench=bench, requirements=requirements, restart_supervisor=restart_supervisor, restart_systemd=restart_systemd, backup= no_backup, force=force, reset=reset)
```

Comment:
For an instance when backup isn't needed, `backup` should be set to `False`
when `--no-backup` flag is used, `no_backup` is set to `True` so `backup = not no_backup` is the proper expression

Reply:
@gavindsouza but in present code I tested if you put --no-backup it will backup that's why I change the logic.

Reply:
This is from testing your branch:

![Screenshot 2020-03-04 at 8 39 26 AM](https://user-images.githubusercontent.com/36654812/75841418-ada61400-5df3-11ea-9125-75f490e1bed4.png)

Reply:
@gavindsouza ok I change the code again